lib/streams/json-parse-stream.js

```diff
@@ -0,0 +1,24 @@
+var Parser = require('jsonparse');
+var ZTransform = require('../transform');
+var util = require('util');
+
+function JSONParseStream (options) {
+  var self = this;
+  this.parser = new Parser();
+
+  if(!options) options = {};
+  if(!options.objectMode) options.objectMode = true;
+  ZTransform.call(this, options);
+
+  this.parser.onValue = function (value){
+    if(this.stack.length === 0) self.push(value);
```

Comment:
I don't think this will work as a JSON parsing stream is expected to work - this will output the entire input JSON object as a single object.  The point of a JSON parsing stream is to be able to parse the object without buffering everything.